sites/en/docs/docs.step

```diff
@@ -24,6 +24,10 @@ site_desc 'message-board', <<-MARKDOWN
 Build a message board! This curriculum is for students who have completed the Suggestotron and the Job Board curricula. This curriculum is a challenge because it won't tell you what to type in!
 MARKDOWN
 
+site_desc 'testing', <<-MARKDOWN
+Increase the stability of your Rails app by learning about tests: what they are, why they're used, and how to use them! This curriculum is for students who have completed the Suggestotron, the Job Board, and the Message Board curricula. There will be challenges!
```

Comment:
It would better for this to have a longer name that is self describing at first glance. Even if a bit redundantly mentioning Rails is still good.  Something like "Testing Rails Applications using RSpec etc..." Then someone can know at a glance what this is about. I am just starting the review process, I am swamped right now, but I will either complete a review tonight or just approve the branch tonight at the latest. This is the one thing I know needs to change now though and it might be the only thing.